Database/QuestieDB.lua

```diff
@@ -38,6 +38,11 @@ local QuestieQuest = QuestieLoader:ImportModule("QuestieQuest")
 ---@type QuestieQuestPrivate
 local _QuestieQuest = QuestieQuest.private
 
+--- A list of quests that will never be available, used to quickly skip quests.
+---@alias AutoBlacklistString "rep"|"skill"|"race"|"class"
+---@type table<number, AutoBlacklistString>
+QuestieDB.autoBlacklist = {}
```

Comment:
This is used in `QuestieDB.lua` and `AvailableQuests.lua`. Not sure if there is a better place for it.

Reply:
Most likely this should be moved to `AvailableQuests.lua`, but only if `QuestieDB.IsDoable` is also moved.